### PR TITLE
Update Web v4 Getting Started guide

### PR DESCRIPTION
**Pull request by JG1995**
Major updates to primarily code examples and restructure of "Guides" menu point.

### CODE REVIEW

File: src/web/v4/getting-started/map.md [hunk 1]
```diff
@@ -34,34 +19,54 @@ eleventyNavigation:
...
 <body>
   <div id="map" style="width: 600px; height: 600px;"></div>
-  ...
+  <script src="main.js"></script>
 </body>
+</html>
 ```
 
-To load data and display it on the map, we need to create a new _instance_ of the [`MapsIndoors` class](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.MapsIndoors.html#MapsIndoors) with a [`mapView` instance](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.mapView.GoogleMapsView.html#GoogleMapsView) with a few _properties_ set:
+To load data and display it on the map, we need to create a new _instance_ of the [`MapsIndoors` class](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.MapsIndoors.html#MapsIndoors) with a [`mapView` instance](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.mapView.GoogleMapsView.html#GoogleMapsView) with a few _properties_ set - This is all done by placing the following code in the `main.js` file you created earlier:
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:69]:
```suggestion
To load data and display it on the map, we need to create a new _instance_ of the [`MapsIndoors` class](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.MapsIndoors.html#MapsIndoors) with a [`mapView` instance](https://app.mapsindoors.com/mapsindoors/js/sdk/latest/docs/mapsindoors.mapView.GoogleMapsView.html#GoogleMapsView) with a few _properties_ set. This is all done by placing the following code in the `main.js` file you created earlier:
```

File: src/web/v4/getting-started/new-project.md [hunk 2]
```diff
@@ -12,16 +12,17 @@ eleventyNavigation:
 {% include "src/shared/getting-started/project/overview.md" %}
 
 <!-- Environment -->
-{% include "src/shared/getting-started/project/environment.md" %}
+## Set Up Your Environment
 
-The project setup is simple. We use pure JavaScript so we do not need to install any frameworks.
+[]: # (The project setup is simple. We use pure JavaScript, so we do not need to install any frameworks. (I don't think we need to mention frameworks, no need to confuse more than neccesary))
```
Comment (markhougaard) [inline src/web/v4/getting-started/new-project.md:18]:
```suggestion
```

File: src/web/v4/getting-started/new-project.md [hunk 3]
```diff
@@ -12,16 +12,17 @@ eleventyNavigation:
...
 
-1. Start by creating a new project folder
+1. While not strictly neccesary, if you do not have prior development experience, it is advised to install an integrated development environment (IDE). A common one to use is [Visual Studio Code](https://code.visualstudio.com/), which is free.
```
Comment (markhougaard) [inline src/web/v4/getting-started/new-project.md:19]:
```suggestion
1. While not strictly neccesary, if you do not have prior development experience, it is advised to install an Integrated Development Environment (IDE). A common one to use is [Visual Studio Code](https://code.visualstudio.com/), which is free.
```

File: src/web/v4/guides/index.md [hunk 4]
```diff
@@ -1,10 +1,25 @@
...
 eleventyNavigation:
   title: Guides
   parent: web-v4
   key: web-v4-guides
   order: 200
----
\ No newline at end of file
+---
+
+You might have come here directly after the "Getting Started" guide, or you might just wish to use these guides at your own pace - Regardless, here is a structured overview over what you can find in these guides. You can also navigate between them in the sidebar to the left.
```
Comment (markhougaard) [inline src/web/v4/guides/index.md:10]:
```suggestion
You might have come here directly after the "Getting Started" guide, or you might just wish to use these guides at your own pace. Regardless, here is a structured overview over what you can find in these guides. You can also navigate between them in the sidebar to the left.
```

**Review by markhougaard — CHANGES_REQUESTED** [review]

File: src/shared/getting-started/overview.md [hunk 5]
```diff
@@ -1,3 +1,3 @@
...
+In this tutorial, you will build your own app from the ground up, gaining experience with the typical development process when working with the MapsIndoors Software Development Kit (SDK). Furthermore, you should be able to gain an understanding of the basic concepts, tools and terminology commonly used when interacting with the SDK. The goal of this guide is to enable you to develop a basic app, that is able to display a map, and incorporate a few basic features such as searching, directions and Live Data integration.
 
-The Live Data relies on a MapsIndoors Dataset which supports Live Data integration, if you do not have access to this we recommend using our demo API key: `{{sdk.tutorialAPIKey}}`.
+Parts of this guide rely on having access to a MapsIndoors Dataset which supports Live Data Integration, if you do not have access to this through your own Solution, we recommend using our demo API key: `{{sdk.tutorialAPIKey}}`.
```
Comment (markhougaard) [inline src/shared/getting-started/overview.md:3]: @JG1995 Reading this again, I'm wondering if we should change some of the terminology used; "Dataset" == "Solution" which in turn is accessed with an "API key". Here's a suggestion:

```suggestion
Parts of this guide rely on having access to a MapsIndoors Solution which supports Live Data Integration. If you do not have access to this through your own Solution, we recommend using our demo API key to access one: `{{sdk.tutorialAPIKey}}`.
```

How does that sound to you?

File: src/web/v4/getting-started/map.md [hunk 6]
```diff
@@ -34,34 +19,54 @@ eleventyNavigation:
 <mi-tab label="MI Components" tab-for="components"></mi-tab>
 
 <mi-tab-panel id="manually">
-The MapsIndoors SDK is hosted on a CDN and should be loaded using a script tag.
+The MapsIndoors SDK is hosted on a content delivery network (CDN) and should be loaded using a script tag.
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:22]:
```suggestion
The MapsIndoors SDK is hosted on a Content Delivery Network (CDN) and should be loaded using a script tag.
```

File: src/web/v4/getting-started/map.md [hunk 7]
```diff
@@ -91,33 +96,51 @@ The MapsIndoors Web Components library can be loaded using [unpkg](https://unpkg
...
+</body>
+</html>
 ```
 
+> Check [@mapsindoors/components](https://www.npmjs.com/package/@mapsindoors/components) for latest version.
+
 After you added the script tag into `<head>`, add the `<mi-map-googlemaps>` custom element into `<body>`. We need to add and populate the `gm-api-key` and `mi-api-key` attributes with your API keys as well:
 
-```html
+<!-- FIGURE OUT HOW TO FIX THE HIGHLIGHTING - IT'S THE <> THAT ARE DOING IT - POTENTIAL FIX MY MAKING IT ONE LINE, IT DOESNT LIKE SPLIT <> -->
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:121]: @JG1995 Remember this comment 🙂

File: src/web/v4/getting-started/map.md [hunk 8]
```diff
@@ -168,13 +200,26 @@ googleMapsInstance.controls[google.maps.ControlPosition.RIGHT_TOP].push(floorSel
 
 Using the `<mi-map-googlemaps>` element, you can add the [floorSelectorControlPosition attribute](https://components.mapsindoors.com/map-googlemaps/) to your existing element. In this case with the value `"TOP_RIGHT"`:
 
-```html
+<!-- FIGURE OUT HOW TO FIX THE HIGHLIGHTING - IT'S THE <> THAT ARE DOING IT - POTENTIAL FIX MY MAKING IT ONE LINE, IT DOESNT LIKE SPLIT <> -->
```
Comment (markhougaard) [inline src/web/v4/getting-started/map.md:203]: @JG1995 Remember this comment

File: src/web/v4/guides/index.md [hunk 9]
```diff
@@ -1,10 +1,25 @@
...
   parent: web-v4
   key: web-v4-guides
   order: 200
----
\ No newline at end of file
+---
+
+You might have come here directly after the "Getting Started" guide, or you might just wish to use these guides at your own pace. Regardless, here is a structured overview over what you can find in these guides. You can also navigate between them in the sidebar to the left.
+
+**NOTE: These guides are for the web-version of the SDK, but we also have extensive documentation for the SDK for iOS and Android.**
```
Comment (markhougaard) [inline src/web/v4/guides/index.md:12]:
```suggestion
**Note: These guides are for the web-version of the SDK, but we also have extensive documentation for the SDK for iOS and Android.**
```

**Review by markhougaard — CHANGES_REQUESTED** [review]

File: src/web/v4/getting-started/map.md (hunk 7 above)
Comment (JG1995) [reply src/web/v4/getting-started/map.md:121]: I know, I made a temporary fix for the time being

Comment (markhougaard) [reply src/web/v4/getting-started/map.md:121]: @JG1995 Can we remove this comment and the other one like it then?

**Review by markhougaard — APPROVED** [review]

**Pull request merged**